quine-cypher/src/main/scala/com/thatdot/quine/compiler/cypher/Functions.scala

```diff
@@ -1485,7 +1485,7 @@ class CypherValueGenFrom(outputType: Type, defaultSize: Long, randGen: (Long, Lo
   }
 }
 
-object CypherGenFroms {
+object CypherGenForms {
```

Comment:
The class is actually called `CypherGenFrom` - these are for Cypher user-defined functions of the form e.g. `gen.int.from`. This object contains implementations of that class.
```suggestion
object CypherGenFroms {
```